metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -194,10 +241,51 @@ def build_corp_user_mce(
             )
         )
 
-        return mce
+    def build_corp_group_mce(
+        self, attrs: dict
+    ) -> Optional[MetadataChangeEvent]:
+        """Creates a MetadataChangeEvent for LDAP groups."""
+
+        if cn := attrs.get("cn"):
```

Comment:
as much as I love the walrus operator, we can't use it since we want to maintain compatibility with Python 3.6+ 🙁